yaptide/routes.py

```diff
@@ -44,13 +44,18 @@ def get():
         errors = shschema.validate(args)
         if errors:
             return errors
-
         param_dict = shschema.load(args)
-        simulation_result = run_shieldhit(param_dict)
+
+        json_data = request.json
+        if not json_data:
+            return {"status": "json error"}
```

Comment:
Add appropriate rest response codes.
https://documentation.commvault.com/11.24/essential/45599_rest_api_response_codes_and_statuses.html

Reply:
That's not what i meant. Look at other endpoints and [this guide](https://github.com/flask-api/flask-api/blob/develop/docs/api-guide/status-codes.md).

Reply:
@retkiewi check it know please